<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenresOptionsPage</class>
 <widget class="QWidget" name="GenresOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>304</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="use_genres">
     <property name="title">
      <string>Use genres from MusicBrainz</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="only_my_genres">
        <property name="text">
         <string>Only use my genres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="artists_genres">
        <property name="text">
         <string>Fall back on album's artists genres if no genres are found for the release or release group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="folksonomy_tags">
        <property name="text">
         <string>Use folksonomy tags as genre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ignore_genres_2">
        <property name="text">
         <string>Genres or folksonomy tags to exclude (comma-separated list):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ignore_genres"/>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Minimal genre usage:</string>
          </property>
          <property name="buddy">
           <cstring>min_genre_usage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_genre_usage">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Maximum number of genres:</string>
          </property>
          <property name="buddy">
           <cstring>min_genre_usage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_genres">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ignore_genres_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Join multiple genres with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="join_genres">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string> / </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>, </string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
